website/src/content/blog/biome-wins-prettier-challenge.md

```diff
@@ -403,3 +403,7 @@ In the next months we will focus on:
 - Publishing a Roadmap. **Keep an eye on it**, it will involve a lot of **interesting** work.
 - Rebranding the website with a new logo.
 - Translate the website in Japanese.
+
+## Translations
+
+- [中文翻译:Biome赢得了Prettier挑战](https://juejin.cn/post/7308643782375768118)
```

Comment:
```suggestion
- [中文翻译: Biome赢得了Prettier挑战](https://juejin.cn/post/7308643782375768118)
```

Is it correct to add a space after `:`? Without the space, it's rendered like this:

![Screenshot 2023-12-04 at 17 13 38](https://github.com/biomejs/biome/assets/602478/b5ec9674-1902-450f-bbbb-ed21d075314f)

It goes on two lines